frontend/test-results/component-integration-Comp-0a7a7-ain-touch-targets-on-mobile-chromium/error-context.md

```diff
@@ -0,0 +1,141 @@
+# Page snapshot
+
+```yaml
```

Comment:
I don't think that anything in the /test-results folder should be checked into Git and be part of the PR